mmv1/products/gkebackup/RestorePlan.yaml

```diff
@@ -496,6 +547,87 @@ properties:
                     description: |
                       A string that specifies the desired value in string format
                       to use for transformation.
+      - !ruby/object:Api::Type::Array
+        name: volumeDataRestorePolicyBindings
+        description: |
+          A table that binds volumes by their scope to a restore policy. Bindings
+          must have a unique scope. Any volumes not scoped in the bindings are
+          subject to the policy defined in volume_data_restore_policy.
+        item_type: !ruby/object:Api::Type::NestedObject
+          required: true
+          properties:
+            - !ruby/object:Api::Type::Enum
+              name: policy
+              required: true
+              description: |
+                Specifies the mechanism to be used to restore this volume data.
+                See https://cloud.google.com/kubernetes-engine/docs/add-on/backup-for-gke/reference/rest/v1/RestoreConfig#VolumeDataRestorePolicy
+                for more information on each policy option.
+              values:
+                - :RESTORE_VOLUME_DATA_FROM_BACKUP
+                - :REUSE_VOLUME_HANDLE_FROM_BACKUP
+                - :NO_VOLUME_DATA_RESTORATION
+            - !ruby/object:Api::Type::Enum
+              name: volumeType
+              required: true
+              description: |
+                The volume type, as determined by the PVC's
+                bound PV, to apply the policy to.
+              exactly_one_of:
+                - restoreConfig.0.volumeType
```

Comment:
Is `exactly_one_of` needed here?

Reply:
Ahh yes this is no needed. Removed. Thank you!